x-pack/solutions/security/packages/navigation/src/i18n_strings.ts

```diff
@@ -8,6 +8,16 @@
 import { i18n } from '@kbn/i18n';
 
 export const i18nStrings = {
+  alertDetections: {
```

Comment:
Super minor but I think we could directly call this one `detections` as it won't have an impact on routing or anything user facing?

Reply:
Yes, it will be just easier to rename all occurrences of `alertDetections` in future.